data/filedefs/filetypes.Kotlin.conf

```diff
@@ -0,0 +1,31 @@
+[styling=C]
+
+[keywords]
+# https://kotlinlang.org/docs/reference/keyword-reference.html
+primary=abstract assert break case catch class const continue default do else enum extends final finally for goto if implements import instanceof interface native new package private protected public return static strictfp super switch synchronized this throw throws transient try volatile while true false null as fun in object typealias val var when by constructor delegate dynamic field file get init param property receiver set setparam where actual annotation companion const crossinline data expect external infix inline inner internal lateinit noinline open operator out reified sealed suspend tailrec vararg field it
+# https://kotlinlang.org/docs/reference/basic-types.html
+secondary=Double Float Long Int Short Byte NaN Void
+# documentation keywords for javadoc
+doccomment=author deprecated exception param return see serial serialData serialField since throws todo version
+typedefs=
+
+[lexer_properties=C]
+
+[settings]
+lexer_filetype=C
+tag_parser=C
+extension=kt
+mime_type=text/x-kotlin
+
+[build-menu]
+FT_00_LB=_Compile Kotlin Class
```

Comment:
@esotericpig Are all Kotlin files class files?

Reply:
A Kotlin file can contain one class or multiple classes, with ".kt" file extension:

https://kotlinlang.org/docs/reference/coding-conventions.html#source-file-names

A Kotlin script file doesn't need a class at all, with ".kts" file extension.

I guess maybe the text should be changed to "Compile Kotlin File"?

Reply:
Thanks. I've changed the menu labels to _Compile, _Execute and Execute _Script - I think this fits better with existing build labels.